charts/datree-admission-webhook/templates/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.Values.rbac.serviceAccount.name}}
+  namespace: {{.Values.namespace}}
+  labels: {{ include "datree.labels" . | nindent 4 }}
+    env: testing
+    owner: datree
+    app: "webhook-server"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels: 
+      app: "webhook-server"
+  template:
+    metadata:
+      labels: {{ include "datree.labels" . | nindent 8 }}
```

Comment:
annotations need to be added

Reply:
NP! But I'm not sure which one. I did add the recommended labels.